tensorflow-sys/build.rs

```diff
@@ -122,7 +124,16 @@ fn install_prebuilt() {
         }); // TODO: remove
 
     println!("cargo:rustc-link-lib=dylib={}", LIBRARY);
-    println!("cargo:rustc-link-search={}", lib_dir.display());
+    let output = PathBuf::from(&get!("OUT_DIR"));
+    let new_library_full_path = output.join(&library_file);
+    if new_library_full_path.exists() {
```

Comment:
Huh, I guess that makes sense.  Let's keep the delete code.  The fewer platform-dependent requirements (like chmod) that we introduce, the better.